### ISSUE TITLE
[Feature Request] Check if this new model could be useful

### ISSUE THREAD

**Issue by Polyterative** [LegendarySpy/Glimpse#97]
### What would you like Glimpse to do?

I found the this new model was released. Maybe we it can be useful for glimpse and improve the quality of the dictation.https://www.epilude.com/news/introducing-model-4

Feel free to close issue if not interested.

### Why would this be useful?

Improved voice dictation, reliability and quality.

### Would this need cloud processing?

None

**Comment by LegendarySpy** [comment]
This is interesting. It looks like it’s a fine-tune based on [Qwen 3 ASR](https://huggingface.co/Qwen/Qwen3-ASR-1.7B). It’s definitely something I’m looking into as an interesting next step for Glimpse—creating custom models. The downside, though, is that creating the proper training data would require people to help train it, and while it could be a community effort, requesting people's dictation data can be taken the wrong way, even if its fully opted in. And with proper privacy masking safe guards.
